src/custom/utils/price.ts

```diff
@@ -1,6 +1,11 @@
 export * from '@src/utils/prices'
 
-const DEFAULT_TIP = '5'
-
 // TODO: add mock API using Alex's created open API definition of tip endpoint
-export const getTip = (): string => DEFAULT_TIP
+export const getTip = async (currencyId: string): Promise<string> =>
+  new Promise(accept => {
+    return setTimeout(() => {
+      console.debug(`[MOCK] utils/price::getTip ==> API request for token ${currencyId}`)
+      const computedTip = Math.floor(Math.random() * 10).toString()
```

Comment:
Be aware this generates a "random" tip amount between 1 - 10 once for each token selected

so it will change sellAmount/buyAmount numbers!
@anxolin @Velenir